docs/classical_control.ipynb

```diff
@@ -130,13 +135,19 @@
     "alice = cirq.NamedQubit('alice')\n",
     "bob = cirq.NamedQubit('bob')\n",
     "message = cirq.NamedQubit('_message')\n",
-    "circuit = cirq.Circuit(\n",
+    "\n",
+    "message_circuit = cirq.Circuit(\n",
```

Comment:
If we're splitting into message and teleport parts, I'd prefer moving the initial Bell state prep into the teleport part, leaving only the operations on `message` in the message circuit.